roles/telemetry_verify_metrics/tasks/verify_mariadb_metrics.yml

```
@@ -0,0 +1,48 @@
+- name: Verify mariadb scrapeconfig exists
+  ansible.builtin.include_role:
+    name: common
+  vars:
+    common_cr_test_id: RHOSO-xxxx
+    common_cr_list:
+      - kind: scrapeconfigs.monitoring.rhobs
+        name: telemetry-mariadb
+
+- name: Get mariadb CR names
+  ansible.builtin.shell: |
+    oc get mariadb -o custom-columns=NAME:.metadata.name --no-headers
```

You'll want to look for a kind "galera" here. Each galera is an instance of the database and each of them has a pod running
```suggestion
    oc get galeras -o custom-columns=NAME:.metadata.name --no-headers
```